department-of-veterans-affairs/caseflow
# Queue | Indicate hearings on other appeals

Resolves #6780

### Description
For vets with a Post Remand and Original appeal, the Post Remand will appear in the user's queue. If the Original appeal had a hearing held, we do not show any indication of that in the Post Remand's Case Details view. This adds text directing the user to View All Cases for more hearing info.

### Acceptance Criteria

### Testing Plan
1. Reseed FACOLS (`rake local:vacols:seed`)
2. Go to `/queue` as `BVAAABSHIRE`, Case Details for Nola L Stroman (994806951) (the case assigned to Abshire is the Post Remand).
3. Scroll to Hearings section, confirm copy appears. Click link, confirm page scrolls to top, View All Cases panel shows

<details><summary>gif</summary>
<img src="https://user-images.githubusercontent.com/8533004/46301103-6028d600-c573-11e8-896c-6877fee1aef9.gif">
</details>
